# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/cairo/cairo/src/cairo-compositor.c
# Commit: 972a07ec6164
# Full Hash: 972a07ec616459f697d835e0ed870ecbf9d822b5
# Author: Jonathan Kew <jkew@mozilla.com>
# Date: 2024-05-07 09:13:07
# Regressor Bug: 1892913
# File Overlap Count: 1
# Description:
#   Bug 1892913 - patch 8 - Update and apply 04-subpixel-aa-api.patch r=gfx-reviewers,lsalzman
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D209415
# ==============================================================================

diff -r dcf06dfaff0c -r 972a07ec6164 gfx/cairo/cairo/src/cairo-compositor.c
--- a/gfx/cairo/cairo/src/cairo-compositor.c	Mon May 06 19:25:16 2024 +0000
+++ b/gfx/cairo/cairo/src/cairo-compositor.c	Mon May 06 19:25:16 2024 +0000
@@ -290,7 +290,8 @@
 	    compositor = compositor->delegate;
 
 	status = compositor->glyphs (compositor, &extents,
-				     scaled_font, glyphs, num_glyphs, overlap);
+				     scaled_font, glyphs, num_glyphs, overlap,
+				     surface->permit_subpixel_antialiasing);
 
 	compositor = compositor->delegate;
     } while (status == CAIRO_INT_STATUS_UNSUPPORTED);